src/vaadin-message-input.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import '@vaadin/vaadin-button/src/vaadin-button.js';
+import '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
+/**
+ * `<vaadin-message-input>` is a Web Component ...
+ */
+class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          align-items: flex-start;
+          box-sizing: border-box;
+          display: flex;
+          max-height: 50vh;
+          overflow: hidden;
+        }
+        vaadin-text-area {
+          align-self: stretch;
+          flex-grow: 1;
+          padding: 0;
+        }
+        vaadin-button {
+          flex-shrink: 0;
+          margin: 0;
+        }
+      </style>
+      <vaadin-text-area placeholder="Message"></vaadin-text-area>
+      <vaadin-button theme="primary contained">Send</vaadin-button>
+    `;
+  }
+
+  ready() {
+    super.ready();
+
+    // Set aria-label to provide an accessible name for the labelless input
+    const textarea = this.shadowRoot.querySelector('vaadin-text-area').inputElement;
+    textarea.setAttribute('aria-label', 'Message');
```

Comment:
+1

Is i18n somewhere documented as ticket? Couldn't find one 

Keeping aria-label and the placeholder of `<vaadin-text-area>` in sync would be good. Exposing aria-label should **not** be needed, when the placeholder is synced with it. (Using placeholder alone isn't enough.. sadly)

Reply:
The i18n API is not designed yet, but we should do it soon. Thanks for the comment! We'll take that into account.

Reply:
Why is the attribute `aria-label="Message"` set in javascript and not in the template?

Reply:
@Peppe because it is targeting the inputElement inside the vaadin-text-area if I'm not mistaking.

Reply:
True. I somehow did not notice the `.inputElement` at the end. Thanks for pointing it out!

Reply:
We will tackle changing the placeholder (and syncing it with aria-label) within #29.